src/frontend/src/lib/utils/network.utils.ts

```diff
@@ -21,6 +24,15 @@ export const isNetworkIdEthereum = (id: NetworkId | undefined): boolean =>
 export const isNetworkIdBitcoin = (id: NetworkId | undefined): boolean =>
 	nonNullish(id) && BITCOIN_NETWORKS_IDS.includes(id);
 
+export const isNetworkIdBTCMainnet = (networkId: NetworkId | undefined): boolean =>
```

Comment:
Unrelated to your PR: I ask my self if we should clean-up the naming, `isNetworkIdBTCMainnet` vs `isNetworkIdBitcoin`

Bitcoin vs BTC. No biggy, just a consistency question that crossed my mind.